First version (#1)
inbo/flandersqmd-revealjs: README.md

**keywords**: quarto; reveal; corporate identity

<!-- community: inbo -->

<!-- description: start -->
This quarto extension builds on the quarto [`revealjs` format](https://quarto.org/docs/presentations/revealjs/) and provides the corporate identity of the Flemish government for slides.
A full example is available on the [GitHub repo](https://github.io/inbo/flandersqmd-revealjs/).
<!-- description: end -->

## Installation

**1.** Start by setting up a quarto project.
You can find more information on how to do this in the [quarto documentation](https://quarto.org/docs/tools/rstudio.html#projects).

**2.** Then open the terminal and go the root of your quarto project.
Run the command below to install the quarto extension and confirm when prompted.
Once installed, you can optionally view the documentation in your browser.

```
quarto install extension inbo/flandersqmd-revealjs
```

**3.** Replace the `project:` section in your `_quarto.yml` file with the following code:

```
project:
  type: default
  preview:
    port: 4201
    browser: true
  render:
    - "*.md"
    - "*.qmd"
    - "!LICENSE.md" #ignore LICENSE.md
    - "!README.md" #ignore README.md
  output-dir: output
```

> **4.** Replace the `format:` section in your `_quarto.yml` file with the following code:

Reviewer: Maybe here add a few sentences on how to control (selected) html settings that are not accessible via section "flandersqmd", but "format" (and can be adapted without breaking things)? E.g., transition-speed?

- `author`: The author(s) of the presentation.
  This is a list of authors.
  Each author is a dictionary with the following keys:
  - `name`: The name of the author.
    This is a dictionary with the following keys:
    - `given`: The given name of the author.
    - `family`: The family name of the author.
  - `email`: The optional email address of the author.
    When set, the author name will be a `mailto` link.
  - `orcid`: The ORCID of the author.
    This if optional.
    Note that INBO requires you to provide an ORCID for all INBO personnel.
  - `affiliation`: The affiliation of the author.
    This is a list of strings.
    One item for each line.
    This is optional.
    Note that INBO requires you to provide at least `Research Institute for Nature and Forest (INBO)` as affiliation for all INBO personnel.
  - `cover`: The background image for the first slide.
    Path to a cover file.
    Should be a `png` or `jpg` file.

### Optional settings

- `subtitle`: The subtitle of the presentation.
- `data`: The date of the presentation.

## Render your quarto presentation

You can render your quarto presentation in two main ways:  

1. **Using RStudio's Build Pane**  

>    - Open your quarto book project in RStudio.

Reviewer: ```suggestion
   - Open your quarto project in RStudio.
```

   - Click on the **Build** tab (usually found in the top-right panel).  

>    - Click **Render Book** to generate all formats.

Reviewer: ```suggestion
   - Click **Render Project** to generate the html presentation.
```

>      You can click on the down arrow next to the **Render Book** button to render only a specific format.

Reviewer: ```suggestion
```

>      If no `flandersqmd-revealjs` format is available, restart your quarto book project and try again.

Reviewer: ```suggestion
```

>    - The output format is saved to disk.

Reviewer: ```suggestion
   - The presentation is saved to disk.
```

2. **Using the Terminal**  
   - Open a terminal and navigate to the folder of your quarto presentation
   - Run the commands below in the terminal.

>      This renders the book and saves it to disk like the **Render Book** button of the **Build** tab.

Reviewer: ```suggestion
     This renders the presentation and saves it to disk like the **Render Project** button of the **Build** tab.
```

>      Preview renders the book and starts a live preview, automatically updating the output when changes are detected.

Reviewer: ```suggestion
     Preview renders the presentation and starts a live preview, automatically updating the output when changes are detected.
```

>      Preview renders the book and starts a live preview, automatically updating the output when changes are detected.

Reviewer: Maybe here add a note that previewing requires the presentation .qmd file to be manually renamed to "index.qmd"?
Per default the presentation .qmd file is named after the quarto project when setting things up.
